docs/source/notebooks_and_ipython/add_kedro_to_a_notebook.md

```diff
@@ -0,0 +1,652 @@
+reviews = pd.read_csv('data/reviews.csv')
+shuttles = pd.read_excel('data/shuttles.xlsx', engine='openpyxl')
+
+# Data processing
+companies["iata_approved"] = companies["iata_approved"] == "t"
+companies["company_rating"] = companies["company_rating"].str.replace("%", "").astype(float)
+shuttles["d_check_complete"] = shuttles["d_check_complete"] == "t"
+shuttles["moon_clearance_complete"] = shuttles["moon_clearance_complete"] == "t"
+shuttles["price"] = shuttles["price"].str.replace("$", "").str.replace(",", "").astype(float)
+rated_shuttles = shuttles.merge(reviews, left_on="id", right_on="shuttle_id")
+model_input_table = rated_shuttles.merge(companies, left_on="company_id", right_on="id")
+model_input_table = model_input_table.dropna()
+model_input_table.head()
+
+# Model evaluation
+
+from sklearn.model_selection import train_test_split
+
+X = model_input_table[[
+    "engines",
+    "passenger_capacity",
+    "crew",
+    "d_check_complete",
+    "moon_clearance_complete",
+    "iata_approved",
+    "company_rating",
+    "review_scores_rating",
+]]
+y = model_input_table["price"]
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=3)
+
+from sklearn.linear_model import LinearRegression
+model = LinearRegression()
+model.fit(X_train, y_train)
+model.predict(X_test)
+from sklearn.metrics import r2_score
+y_pred = model.predict(X_test)
+r2_score(y_test, y_pred)
+```
+
+## Use Kedro for data processing
+Even if you’re not ready for a full Kedro project, you can still take advantage of its data handling solution in your existing project from within a notebook. This section shows you how.
+
+Kedro’s Data Catalog is a registry of all data sources available for use by the project, and offers a separate place to declare details of the datasets your projects use. Kedro provides [built-in datasets for numerous file types and file systems](https://docs.kedro.org/en/stable/kedro_datasets.html), so you don’t have to write any of the logic for reading or writing data. 
+
+Kedro offers a range of datasets, including CSV, Excel, Parquet, Feather, HDF5, JSON, Pickle, SQL Tables, SQL Queries, Spark DataFrames and more. They are supported with the APIs of pandas, spark, networkx, matplotlib, yaml and more. It relies on `[fsspec](https://filesystem-spec.readthedocs.io/en/latest/)` to read and save data from a variety of data stores including local file systems, network file systems, cloud object stores, and Hadoop. You can pass arguments in to load and save operations, and use versioning and credentials for data access.
+
+To start using the Data Catalog, create a `catalog.yml` in the same folder as your notebook to define datasets that can be used when writing your functions. For example:
+
+
+<!--This code needs the user to create a yaml file with this contents. Is there a piece of code we could offer that creates and writes one for them into the appropriate directory to save the reader the manual task? -->
```

Comment:
Yep, agreed. I think we should clone/copy/download the folder as suggested. I want to keep it in the same repo though to ensure that updates are synchronised rather than need two different PRs in different places. Also the `kedro-academy` repo is a bit like the wild west compared to this Kedro repo, and we probably should have an `examples` folder in docs for other example code (should we ever make it).